test/Microsoft.AspNetCore.Diagnostics.FunctionalTests/MiddlewareAnalysisSampleTest.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
It's testing logging the pipeline structure itself, which none of the others do.